src/components/card/CardSidebarTabAttachments.vue

```diff
@@ -136,15 +154,24 @@ export default {
 		this.$store.dispatch('fetchAttachments', this.card.id)
 	},
 	methods: {
+		handleDropFiles(event) {
```

Comment:
Ideally we should move this handling into a mixin and the overlay into a separate component, since then we can reuse this for the `CardItem` where dragging files to should also work. But feel free to just finish this for the sidebar first and then we do the card handling in another pr.

Reply:
let's do it in another PR.
Could you do a quick review, please? And merge if it's ok.

Reply:
Added to the list of https://github.com/nextcloud/deck/issues/1597